.gitignore

```diff
@@ -28,3 +28,4 @@ cabal.sandbox.config
 
 # IDE project files
 .idea
+app/Svg/Generator.hs
```

Comment:
@frankjohnlee please revert this change. This is not appropriate (either for this specific pull request, or in general).